src/js/components/DeploymentComponent.jsx

```diff
@@ -81,6 +149,35 @@ var DeploymentComponent = React.createClass({
     }
   },
 
+  onContinueMigrationSuccess: function (response, appId) {
+    var continueButtonsLoadingState = Object.assign({},
+      this.state.continueButtonsLoadingState);
+    continueButtonsLoadingState[appId] = false;
+    this.setState({
+      continueButtonsLoadingState
+    }, function () {
+      DialogActions.alert({
```

Comment:
Yeah I know, but I don't think it gets any more readable with such long arguments. I'll keep these as they are.